src/cic/repl.clj

```diff
@@ -15,33 +17,68 @@
 
 (defn write-projection-tsv
   [outfile projection]
-  (let [fields (juxt (comp date->str :date) :actual :min :q1 :median :q3 :max)
-        headers ["date" "actual" "min" "q1" "median" "q3" "max"]]
+  (let [fields (juxt (comp date->str :date)
```

Comment:
and I think these output functions will want to go into their own ns's at some point too.